sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java

```diff
@@ -41,18 +39,20 @@ public static DirectConnectionConfig getDefaultConfig() {
         return new DirectConnectionConfig();
     }
 
-    //  TODO: (DANOBLE) - To update these docs.
-
     /**
-     * Gets the direct connection timeout
+     * Gets the connection timeout for direct client,
```

Comment:
Is 'direct client' mean Direct TCP connection? 
Client & connection overload confusing.

Reply:
direct client means direct tcp client itself.